### DISCUSSION TITLE
fontawesome 样式没有加载

### DISCUSSION THREAD

**Discussion by lym210119** [spencerwooo/onedrive-vercel-index#610] [Q&A] [answered]
是不是没有引入 import '@fortawesome/fontawesome-svg-core/styles.css'

**Comment by spencerwooo** [comment, marked as answer]
什么叫 fontawesome 样式没有加载，演示站的 icon 显示不是预期情况吗？另外，我在官方文档上并没有看到需要单独加载这段 CSS 的配置说明。

**Reply by hgez6** [reply]
好像只能加载fontawesome 样式里brand类型的，其他类型似乎无法加载图标

**Reply by spencerwooo** [reply]
是这样的，具体：

https://github.com/spencerwooo/onedrive-vercel-index/blob/26945f554e7c40787cce3103756174b1fe7afe0d/pages/_app.tsx#L62-L64

React 中引入 FontAwesome 是按需引入的，为了让 Navbar 能自定义图标我才把所有 brand icons 引入。
